source/context_menus/_contextMenuDictionary.js

```diff
@@ -0,0 +1,26 @@
+const { ContextMenuWrapper } = require("../classes");
+
+/** @type {string[]} */
+exports.contextMenuFiles = [
+	"_context_menu_stats.js"
```

Comment:
File name convention questions:
- Was there a reason for the leading underscore in the stats filename? I've been using that to sort "important" files to the top of the directory
- Confirming: the intent was to keep both user context menu options and message context menu options in the same dictionary, accepting the uniqueness constraint created within those two categories?